yarn.lock

```diff
@@ -1005,10 +1005,10 @@
   resolved "https://registry.yarnpkg.com/@justeat/fozzie-colour-palette/-/fozzie-colour-palette-2.5.0.tgz#a26555e476c6103cdcd9fdf233238af6b0321cee"
   integrity sha512-qMWN2jSOhV/d0t6lmDnN4ae8n5pKNgXvq/BNM3q/t2RZz4Zckzxdy/g+H/tVsqYP6o8Liu1e05VXbpwAqTKeWQ==
 
-"@justeat/gulp-build-fozzie@10.0.0":
-  version "10.0.0"
-  resolved "https://registry.yarnpkg.com/@justeat/gulp-build-fozzie/-/gulp-build-fozzie-10.0.0.tgz#51e673916bd95ab49198f5216fb5f8694ce9eb7c"
-  integrity sha512-bky22uG6nYwgtqV4F/UTt7Ix5tbLEtoZoRxEhdMsHtD8mMD1wEoawi04PO0jaP7oA5+s5TkQH0j9Coh5MtFXPQ==
+"@justeat/gulp-build-fozzie@10.1.0":
+  version "10.1.0"
+  resolved "http://packages.je-labs.com/npm/private-npm/@justeat/gulp-build-fozzie/-/gulp-build-fozzie-10.1.0.tgz#2b59ba67d0457e1aafdf64ab9031418dd5b7a8c7"
```

Comment:
You need to switch to the public yarn registry feed otherwise Travis can't resolve this package. You can set up npmrc on the command line to handle switching between profiles to help with this type of thing.